cms/djangoapps/contentstore/views/course.py

```diff
@@ -469,6 +472,56 @@ def _get_rerun_link_for_item(course_key):
     return reverse_course_url('course_rerun_handler', course_key)
 
 
+def _ora1_deprecation_info(course_id, advanced_modules):
+    """
+    Returns information about ORA1
+        * Modules presence in the Advanced Module List.
+        * Components presence in the course outline.
+
+    Arguments:
+        course_id (CourseKey): course id
+        advanced_modules (list): advance module list
+
+    Returns:
+        Dict with following keys:
+
+        ora1_enabled (bool): True if "peergrading" and/or "combinedopenended" are in the Advanced Module List else False
+        ora1_components (list): List of ora1 components and their parent's url
+        advance_settings_url (str): URL to advance settings page
+    """
+    data = {
+        'ora1_enabled': 'peergrading' in advanced_modules or 'combinedopenended' in advanced_modules,
+        'ora1_components': [],
+        'advance_settings_url': reverse_course_url('advanced_settings_handler', course_id)
+    }
+
+    cache_key = 'ora1.components.{course}'.format(course=course_id)
+
+    ora1_components = cache.get(cache_key)  # pylint: disable=maybe-no-member
+    if not ora1_components:
```

Comment:
I think you want to do an explicit check `is not None` here. Otherwise, you'll incur the extra work even if it's an empty list (since that is falsy in Python).